src/gui/sdlmain.cpp

```diff
@@ -823,6 +823,28 @@ static const VsyncPreference &get_vsync_preference()
 	                              : sdl.vsync.when_windowed;
 }
 
+static void set_vfr_backfill_tempo_from_rate(const double dos_rate_hz)
+{
+	constexpr auto min_dos_rate_hz    = 24.0;
+	constexpr auto max_dos_rate_hz    = 1000.0;
+
+	constexpr auto min_backfill_tempo = 4.0;
+	constexpr auto max_backfill_tempo = 18.0;
+
+	const auto tempo = remap(min_dos_rate_hz,
+	                         max_dos_rate_hz,
+	                         min_backfill_tempo,
+	                         max_backfill_tempo,
+	                         dos_rate_hz);
+
+	LOG_MSG("SDL: Setting VFR backfill tempo to %.0f "
+	        "from a DOS rate of %.1f Hz ",
```

Comment:
"SDL: Setting VFR backfill _rate_ to %.0f _Hz_ from a DOS rate of %.1f Hz"

Reply:
I included this log if @0mnicydle was interested in making adjustments; but I think we're all set now, so commented it out. It's there if we really need to dig into it again, but otherwise it's pretty noisy and logs every video mode and window change (when DOS rate might change).

```c++
// LOG_MSG("SDL: Setting VFR duplicate countdown to %.0f "
//         "from a DOS rate of %.1f Hz ", countdown, dos_rate_hz);
```

I didn't postfix the countdown variable with Hz because `countdown` isn't a per-second rate. It's just a simple decrement-and-trigger ("N calls of the VFR rendering function after which a  duplicate frame is presented").